tests/integration/test_interaction.py

```diff
@@ -284,6 +295,62 @@ def get_locations():
             self.assertDictEqual(locations_after_reset[item_key], initial_locations[item_key])
             self.assert_reverted_item(item_key)
 
+    def alt_text_image_displayed(self, items_map):
+        #Verify that the alt text for the underlying image appears when the user navigates to it with the keyboard.
+        target_img = self._page.find_element_by_css_selector('.target-img')
+        alt_text = target_img.get_attribute('alt')
+
+        # Navigate to dropzone using keyboard
+        items_container = self._page.find_element_by_css_selector('.target')
+
+        # Check that text that appears is the same as the alt-text
```

Comment:
@dovinmu nit: this comment is somewhat unnecessary - it just describes what next line do, which is kind of clear even without this comment.